latch/resources/workflow.py

```diff
@@ -1,95 +1,94 @@
 import inspect
 from dataclasses import is_dataclass
 from textwrap import dedent
-from typing import Callable, Union, get_args, get_origin
+from typing import Callable, Dict, Optional, Union, get_args, get_origin
 
 from flytekit import workflow as _workflow
 from flytekit.core.workflow import PythonFunctionWorkflow
 
-from latch.types.metadata import LatchMetadata
+from latch.types.metadata import LatchAuthor, LatchMetadata, LatchParameter
+
+
+def _generate_params(params: Dict[str, inspect.Parameter]) -> Dict[str, LatchParameter]:
+    return {param: LatchParameter(display_name=param) for param in params}
```

Comment:
```suggestion
    return {param: LatchParameter(display_name=best_effort_display_name(param)) for param in params}
```

would do this below too

Reply:
also maybe `batch_table_column=True`